pythran/toolchain.py

```diff
@@ -153,56 +153,21 @@ def __str__(self):
         specs = expand_specs(specs)
         check_specs(ir, specs, renamings)
 
-        mod = BoostPythonModule(module_name, docstrings)
-        mod.use_private_namespace = False
-        # very low value for max_arity leads to various bugs
-        min_val = 2
-        specs_max = [max(map(len, s)) for s in specs.itervalues()]
-        max_arity = max([min_val] + specs_max)
-        mod.add_to_preamble(Define("BOOST_PYTHON_MAX_ARITY", max_arity),
-                            Define("BOOST_SIMD_NO_STRICT_ALIASING", "1"))
+        metainfo = {'hash': hashlib.sha256(code).hexdigest(),
+                    'version': __version__,
+                    'date': datetime.now(),
+                    }
+
+        mod = PythonModule(module_name, docstrings, metainfo)
+        mod.add_to_preamble(Define("BOOST_SIMD_NO_STRICT_ALIASING", "1"))
         mod.add_to_includes(Include("pythonic/core.hpp"),
                             Include("pythonic/python/core.hpp"),
+                            Include("pythonic/types/bool.hpp"),
+                            Include("pythonic/types/int.hpp"),
```

Comment:
OK, can you just add a fixme? I will certainly continue on the include stuff so I may work on it once it is merged.